docs/index.md

```diff
@@ -76,19 +76,20 @@ Those with data access, who contribute to the project, including data management
 From [the Center for Humanities Computing at Aarhus University](https://chc.au.dk/):
 
   - Kenneth Enevoldsen ([kenneth.enevoldsen@cas.au.dk](mailto:kenneth.enevoldsen@cas.au.dk))
-  - Peter Vahlstrup ([imvpbv@cc.au.dk]((mailto:imvpbv@cc.au.dk))
-  - Per Møldrup-Dalum ([per@cas.au.dk]((mailto:per@cas.au.dk))
-  - Kristoffer Laigaard Nielbo ([kln@cas.au.dk]((mailto:kln@cas.au.dk))
-  - Jan Kostkan ([jan.kostkan@cas.au.dk]((mailto:jan.kostkan@cas.au.dk))
+  - Marton Kardos
```

Comment:
No email for Marton?

Reply:
Hmm in general I was considering if we want emails on everyone.

We had a few cases in the HOPE project, where we got a few cases of hateful and threatening emails to the entire team, but I see that as it stands he is the only one without an email so I have just added it for now:

 ```suggestion
  - Marton Kardos ([martonkardos@cas.au.dk](mailto:martonkardos@cas.au.dk))
```